git-clone-repos.yaml

```diff
@@ -49,20 +52,24 @@
       shell: |
         cd ~/{{ DIR_PATH }}/{{ item }}
         git remote rename origin upstream
-        git remote add origin git@github.com:{{ USER }}/{{ item }}.git
+        git remote add origin git@github.com:{{ GITHUB_USER }}/{{ item }}.git
```

Comment:
so its not the user on the local workstation, its the username on github?!

Reply:
yes